### DISCUSSION TITLE
Biolink definitions for clinical edge attributes

### DISCUSSION THREAD

**Discussion by karafecho** [NCATSTranslator/Clinical-Data-Committee-Tracking-Voting#26] [General]
The TCDC has standardized on the following edge attributes for the TCDC Curated Query Service: log_odds_ratio, log_odds_ratio_95_ci, total_sample_size, infores id, URL to wiki page. We now need to define the first three attributes in Biolink Model. I have proposed definitions below. These are adapted from the descriptions provided in https://www.ncbi.nlm.nih.gov/books/NBK431098/, just so we have supporting evidence in whatever definitions we agree upon.

This discussion thread is intended to stimulate comment and reach consensus. **Please comment on or before Friday, April 28.**

**Comment by karafecho** [comment]
log_odds_ratio: The logarithm of the odds ratio, or the ratio of the odds of event Y occurring in an exposed group versus the odds of event Y occurring in a non-exposed group.

**Updated, more explicit definition, 07.07.2023:**

log_odds_ratio: The natural logarithm of the odds ratio, or the ratio of the odds of event Y occurring in an exposed group versus the odds of event Y occurring in a non-exposed group.

**Reply by karafecho** [reply]
@sierra-moxon : Please see the above updated definition. I can reopen the ticket, if you can help me find it. :-)

**Reply by nlharris** [reply]
this may be the ticket: https://github.com/biolink/biolink-model/issues/1301

**Reply by karafecho** [reply]
Thanks, Nomi!

**Comment by karafecho** [comment]
log_odds_ratio_95_ci: The ninety-five percent confidence range in which the true log odds ratio for the sample population falls.

**Reply by CaseyTa** [reply]
COHD by default calculates 99% CI to reduce the false-positives rate given the potentially large number of edges. If others are giving 95%, then we provide that also (we'll probably do it in addition to 99% CI if we can do so unambiguously).

**Reply by karafecho** [reply]
ICEES KG provides 95% CI.

**Reply by GitHubbit** [reply]
Multiomics EHR Risk is fine with whatever alpha the committee decides on, since we have yet to implement this. That said, I believe the packages we use default to 5%...
-Kamileh

**Reply by CaseyTa** [reply]
Sounds good, we will comply with 95.

**Reply by karafecho** [reply]
FWIW, I think it's fine to keep the 99% CI, too. While we agreed on a small set of edge attributes with standardized names, I don't think anything will break by having additional edge attributes.

**Comment by karafecho** [comment]
total_sample_size: The total number of patients or participants within a sample population.

**Comment by karafecho** [comment]
**Consensus reached, 04.28.2023**
